[IS-5.11.0] Document on resend-code API changes for mobile claim update (#3091)
wso2/docs-is: en/docs/develop/enable-verification-for-updated-mobile-number.md

        enable_verification = true
        verification_sms_otp_validity = “5”
        ```

>     3. By default mobile number verification is not allowed for privileged users. Add the following property to the above `deployment.toml` config to enable this server wide.

Reviewer: ```suggestion
    3. By default, mobile number verification is not allowed for privileged users. Add the following property to the above `deployment.toml` config to enable this server wide.
```

        ```toml
        [identity_mgt.user_claim_update.mobile]
        enable_verification_by_privileged_user = true
        ```

>     4. Add the following properties to the `deployment.toml` file to restrict resending verification codes only by privileged users.

Reviewer: ```suggestion
    4. Add the following properties to the `deployment.toml` file to restrict resending verification codes for privileged users.
```

        ```toml
        [[resource.access_control]]
        context = "(.*)/api/identity/user/v1.0/resend-code(.*)"
        secure = "true"
        http_method = "all"
        permissions=["/permission/admin/manage/identity/identitymgt"]
        scopes=["internal_identity_mgt_view","internal_identity_mgt_update","internal_identity_mgt_create","internal_identity_mgt_delete"]
        ```

>         The mobile number verification by a privileged user is available as an update in WSO2 IS 5.11.0 from update level 150 onwards (Updates 2.0 model). If you don't already have this update, see the instructions on [updating WSO2 products](https://updates.docs.wso2.com/en/latest/updates/overview/).

Reviewer: !!! info
    Mobile number verification by a privileged user is available as an update in WSO2 IS 5.11.0 from update level 150 onwards (Updates 2.0 model). If you don't already have this update, see the instructions on [updating WSO2 products](https://updates.docs.wso2.com/en/latest/updates/overview/).

Reviewer: Let't put this to the beginning of the note

Author: Addressed with d9428901276377ae3f901fdab573f5006f7846ff

"HTTP/1.1 201 Created"
```

> Additionally can send the following curl command to resend a new SMS OTP code by a privileged user.

Reviewer: ```suggestion
Additionally, you can use the following curl command to resend a new SMS OTP code by a privileged user.
```

Reviewer: The note about the WSO2 update should be added here

Author: Addressed with d9428901276377ae3f901fdab573f5006f7846ff

**Sample**

!!! abstract ""
    **Request**
    ```curl
    curl -X POST -H "Authorization: Basic <Base64Encoded_username:password>" -H "Content-Type: application/json" -d '{"user":{},"properties": []}'
    "https://localhost:9443/api/identity/user/v1.0/resend-code"
    ```

    The user and the verification scenario should be specified in the request body as follows :
    ```
    "user": {"username": "", "realm": ""}
    "properties": [{"key":"RecoveryScenario", "value":"MOBILE_VERIFICATION_ON_UPDATE"}]}
    ```
    ---
    **Sample Request**
    ```curl
    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user":{"username": "admin","realm": "PRIMARY"},"properties": [{"key":"RecoveryScenario","value":"MOBILE_VERIFICATION_ON_UPDATE"}]}' "https://localhost:9443/api/identity/user/v1.0/resend-code" -k -v
    ```
    ---
    **Response**
    ```
    "HTTP/1.1 201 Created"
    ```

> !!! Note

Reviewer: Seems to be duplicated. The same note was available at the top.
We need to put this notification at the beginning before the user sets out to enable these configurations.

Author: Actually this was referring to two updates. Configuration is only required for the mobile number verification. This update is about resending the OTP
